packages/datadog/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "lib": [
-      "es2022"
+      "es2022",
+      "dom"
```

Comment:
Not keen on having `dom` in everything that includes steveo. Did you try to declare the missing ReadableStream ?

```
declare global {
     interface ReadableStream {}
 }
```

Reply:
Done @sugendran.
The ReadableStream interface was declared globally for all the packages that were not compiling. The `dom` configuration was removed.